build/azure-pipelines/darwin/continuous-build-darwin.yml

```diff
@@ -60,3 +60,6 @@ steps:
     testResultsFiles: '*-results.xml'
     searchFolder: '$(Build.ArtifactStagingDirectory)/test-results'
   condition: succeededOrFailed()
+- task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0 # {{SQL CARBON EDIT}} add task
```

Comment:
Doesn't this get automatically injected anyway?

Reply:
Kind of, it used to be auto inject into the pipeline (like you could actually see the step) now it looks like it runs silently in the background. I'm seeing how useful actualyl seeing the step is.

Reply:
My hope is that it follows the same settings as in the component governance tab and that this step could actually be interesting to fail on for CI.